products/idn/tools/cli/index.md

```diff
@@ -0,0 +1,224 @@
+New-Item -ItemType Directory -Path 'C:\Users\<username>\.sailpoint'
+New-Item -ItemType File -Path 'C:\Users\<username>\.sailpoint\config.yaml'
+```
+These commands will create a `config.yaml` file you can use to set your environment variables:
+
+The `config.yaml` file must contain the following information:
+
+```yaml
+authtype: pat # Can be either pat or oauth, both methods can be configured, with only one being active at a time.
+debug: false # Setting debug to true, will result in more verbose output
+oauth: # All OAuth specific configuration information
+  authurl: https://example.identitynow.com/oauth/authorize
+  baseurl: https://example.api.identitynow.com
+  clientid: example-client-id
+  clientsecret: ""
+  redirect:
+    path: /callback
+    port: 3000
+  tenant: example
+  token:
+    accesstoken: example-access-token
+    expiry: example-token-expiry-date
+  tokenurl: https://example.api.identitynow.com/oauth/token
+pat: # All Personal Access Token specific configuration information
+  tenant: example
+  baseurl: https://example.api.identitynow.com
+  tokenurl: https://example.api.identitynow.com/oauth/token
+  clientsecret: example-client-secret
+  clientid: example-client-id
+  token:
+    accesstoken: example-access-token
+    expiry: example-token-expiry-date
+```
+You can copy the example into your `config.yaml` file. You must specify your baseurl, tokenurl, clientsecret, and clientid and any other necessary information for your chosen authentication method. 
+
+You can have both authentication methods configured at once, but only one can be active at a time. 
+
+#### Environment variable configuration 
+
+You can also store your configuration in environment variables. 
+On **Linux/Mac**, open your terminal app and run these commands to export the following environment variables:
+
+```shell
+export SAIL_BASEURL=https://{tenant}.api.identitynow.com
+export SAIL_TOKENURL=https://{tenant}.api.identitynow.com/oauth/token
+export SAIL_CLIENTID={clientID}
+export SAIL_CLIENTSECRET={clientSecret}
+```
+
+To get your environment variables to persist across terminal sessions, add these exports to your shell profile, something like `~/.bash_profile`.
+
+On **Windows**, run Powershell as an administrator and run the following commands: 
+
+```powershell
+$env:SAIL_BASEURL = 'https://{tenant}.api.identitynow.com'
+$env:SAIL_TOKENURL = 'https://{tenant}.api.identitynow.com/oauth/token'
+$env:SAIL_CLIENTID = '{clientID}'
+$env:SAIL_CLIENTSECRET = '{clientSecret}'
+```
+
+To get your environment variables to persist across PowerShell sessions, run this command instead: 
+
+```powershell
+[System.Environment]::SetEnvironmentVariable('SAIL_BASEURL','https://{tenant}.api.identitynow.com')
+```
+
+## Usage
+
+Run the `sail` command for an overview of the available commands and flags. You can use the `-h` flag with any available command to see additional available options for each command: 
+
+The following commands are available.: 
```

Comment:
We have more commands, I'm currently in the process of creating these nested files for them, whats the best way for me to get that content to you?

Reply:
Okay nice, you can just message me and I'll add it in, or keep updating the readme and I'll be better about checking it, or you can update my branch directly and throw them in there, and I'll edit them. Whatever is easiest for you works for me.